src/core/logic.py

```diff
@@ -1648,7 +1648,7 @@ def send_proposal_change_owner_ack(request, proposal, email_text, owner):
 
 
 def send_task_decline(assignment, _type, email_text, sender, request):
-    if request and request.user.is_authenticated:
+    if request and request.user.is_authenticated():
```

Comment:
Ii `if request` is necessary? 
`request` isn't an optional argument - are there cases where this function is called with `request=None`?